packages/story-editor/src/components/panels/document/slug/test/slug.js

```diff
@@ -76,6 +77,13 @@ describe('SlugPanel', () => {
     expect(url).toBeInTheDocument();
   });
 
+  it('should not display the input when using non-pretty permalinks', () => {
```

Comment:
I think the proper term is [Unpretty](https://www.youtube.com/watch?v=g2gy1Evb1Kg) 😂

Reply:
Perhaps, it didn't sound nice though 😛